### DISCUSSION TITLE
Getting unexpected token export when compiling after npm install date-fns

### DISCUSSION THREAD

**Discussion by juanma1331** [vercel/next.js#17148] [Help] [answered]
Node v14.10.0
Npm v6.14.8
Next v9.5.3
Hi, after installing date-fns using npm im getting this error. I did not find anything relative to this.
Some clue would be really appreciated, is my first time using the framework.

![error-console](https://user-images.githubusercontent.com/46537115/93375527-9216ce00-f858-11ea-9667-284e5c1531a1.png)

![error-web](https://user-images.githubusercontent.com/46537115/93375546-98a54580-f858-11ea-8c59-7077d8b249b6.png)

**Comment by jamesmosier** [comment, marked as answer]
Next does not support esm imports. You could transpile it with https://github.com/martpie/next-transpile-modules although I use date-fns with no issues 🤔
